proto/viam/app/packages/v1/packages.proto

```diff
@@ -83,6 +83,18 @@ message DeletePackageResponse {
 message Package {
   PackageInfo info = 1;
   string uri = 2;
+  google.protobuf.Timestamp created_on = 3;
+}
+
+message UploadedPackage {
```

Comment:
Can you add a comment that this is what's stored in the DB?

Reply:
If the difference between Package and UploadedPackage is essentially that UploadedPackage is our internal view, I wonder if it should have a name indicating that. Might be worth asking Eric if there's a generally accepted pattern to follow here? Non blocking, but worth checking imo

Reply:
done!

Reply:
+1 I think InternalPackage would be a good rename, and include that wording of "our internal view" in the comment.

We'll include Eric in the force-merge to remove created_on so can additionally get input there — want this merge to be unblocked for now.

Reply:
updated